_Group Projects/Group_2/Week_5/Readme.md

```diff
@@ -0,0 +1,21 @@
+Please change the working directory to the file path that goes to the Box folder on your own computer when running the demonstration do file
+
+This program will help you determine if you should run a linear regression model on your dataset, as well as let you know of any outliers.
+
+
+After you loaded the dataset, the syntax is 
+
+ . myprogram dep_var ind_var_1 ... ind_var n
+
+Where the dep_var is your variable of interest for the potential regression and all ind_var_k variables are independent variables that might have outliers.
+
+The program first drops any string variables and identifies the dependent variable as the first variable you entered.
+
+Next, it generates a covariance matrix, which is useful to assess collinearity issues, and then it regresses the dependent variable on each non-string variable in the dataset in order to construct the error plots - which are saved under varname.gph
```

Comment:
You may want to think more about the scope of your program, as you're trying to accomplish a lot of checks with one command. Though all of this information may be useful in the model selection process, a more compact and focused program output is often preferable. When your program outputs many tables and graphs, it can lose some of its utility as users scroll through the results trying to understand what everything means and how it relates to their model selection decision. For example, the covariance matrix feels a tad out of place here, as the other aspects of the program are more relevant to the goal of assessing standard errors and outliers. Checking for collinearity is certainly important when constructing a model, but is it necessary for this program?